File: Day-5/program3.cpp
#include<iostream>
using namespace std;
int main()
{//opening main
    int n;//initialising variables
    cout<<"ENTER NUMBER";
    cin>>n;//input number from keyboard
    for(int i =1;i<=n;i++)
    {//opening for loop
        if(n%i==0)
        cout<<i<<"\n";//printing factors of number
    }//closing for loop
    return 0;
}//closing main

File: Day-13/program3.java
import java.util.Scanner;
public class program3
{//opening class
    public static void main(String[] args)
     {//opening main
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter size of array=");
        int n=sc.nextInt();//initialising variables
        int ar[]=new int[n];
        System.out.println("Enter array elements=");
        for(int i=0;i<n;i++)
        {//opening for loop
            ar[i]=sc.nextInt();
        }//closing for loop
        int lar=ar[0];
        int sma=ar[0];
        for(int i=1;i<n;i++) 
        {//opening for loop
            if(ar[i]>lar) 
            {
                lar=ar[i];
            }
            if(ar[i]<sma)
            {
                sma=ar[i];
            }
        }//closing for loop
        System.out.println("Largest element of array="+lar);
        System.out.println("Smallest element of array ="+sma);
        sc.close();
    }//closing main
}//closing class